City-of-Pori/pori-kada
# PORI-279: Unlimited project-liftups.

Changed project-liftups to be unlimited. Small style tweaks as well.

To test:
1. drush fra -y && drush cc all
2. Create multiple liftups with project flag.
3. Confirm that there are more than 2 liftups visible (for example on the front page if you set the setting to publish on frontpage)
